Tasks/ServiceFabricComposeDeploy/task.json

```diff
@@ -15,8 +15,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 0,
-        "Patch": 39
```

Comment:
Why were these different between task.json and task.loc.json, but now they are the same?   That is probably good, but why the jump from 0.0.39 to 0.1.1?